inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -322,7 +346,31 @@ int main(int argc, char *argv[]) {
         std::string topology_name = "";
         benchmark_app::InputsInfo app_inputs_info;
         std::string output_name;
-        if (!isNetworkCompiled) {
+
+        if (!FLAGS_cache.empty()) {
```

Comment:
benchmark supports 2 way to set parameters - using config json file or using command line parameters. Command line parameters has a first priority. So please handle this new option appropriately (see SetConfig above).